### ISSUE TITLE
Check for Backspace pressing to prevent unneeded redirect to "Open Project" screen

### ISSUE THREAD

**Issue by Opennat** [OpenRefine/OpenRefine#1025]
When project is open, Refine should check pressing of Backspace button, like it work in Wikipedia and many other pages ("Really want to leave this page?"). It should control Backspace, when cursor focus does not set in any editable field.

Reason: sometimes you click in editable field (for text filter or transforming options, as sample), and for some reason: network or interface lags or you just accidentally slightly touch touchpad to unfocus, this field does not gets / lost focus (you could not be shure, as the cursor does not blinking in Refine), then pressing Backspace to edit data, and voila - start over.

That is really annoying, when you work fast enough.

OpenRefine now does not have ability to save project state (except of filters set via plugin that uses GET strings to control this), so it will be simple insurance option not to lose time and/or work.

**Comment by jackyq2015** [comment]
Noted. Will check how to implement this.

**Comment by tfmorris** [comment]
That's a good suggestion, but, just for clarification, all project state except for facets is saved automatically, so nothing should be lost.  If you want to save a particular facet setup, you can just click on the permalink at the top of the screen.  No plugins are required.

**Comment by Opennat** [comment]
True, but when you have project with many facets(/filters) number - you need to keep in mind them all time, then set them up again, or constantly (at any change of them) use "permalink".
